.github/workflows/test.yml

```diff
@@ -39,4 +41,4 @@ jobs:
         run: ssh default 'sh -exc "uname -a && df -T"'
 
       - name: tests
-        run: ssh default 'cd /vagrant && make test'
+        run: ssh default 'cd /vagrant && make test && make test-local'
```

Comment:
can be just `make test test-local`

Reply:
...but rather than having a need to call `test-local` separately, I'd add `test: test-local` dependency into the Makefile.

Reply:
Yes, makes sense; tests run really fast, so shouldn't be an issue to always run both